e2e/specs/openConceptLab.spec.ts

```diff
@@ -13,22 +13,26 @@ test.beforeEach(async ({ api }) => {
 test('should be able to setup a subscription and import concepts', async ({ page }) => {
   const openConceptLabPage = new OpenConceptLabPage(page);
 
-  // Setup the subscription
+  await test.step('When I setup the subscription', async () => {
   await openConceptLabPage.goto();
   await openConceptLabPage.addOclSubscription();
+  });
 
-  // Start an Import
+  await test.step('And I starts and import', async () => {
   await openConceptLabPage.importTab().click();
   await openConceptLabPage.startImport();
+  });
 
-  // Check results of the import
+  await test.step('Then the import results should show in the test report', async () => {
   await openConceptLabPage.goto();
   await openConceptLabPage.previousImportsTab().click();
   await expect(openConceptLabPage.previousImportsTable()).toHaveText(/\d+ items fetched/);
+  });
 
-  // Unsubscribe
+  await test.step('And I unsubscribes', async () => {
   await openConceptLabPage.subscriptionTab().click();
   await openConceptLabPage.unsubscribe();
+  });
```

Comment:
I think it's better to move this block into an `afterEach` block. (maybe with another ticket)

```suggestion
  await openConceptLabPage.subscriptionTab().click();
  await openConceptLabPage.unsubscribe();
```

Reply:
This is not needed since there's only one test case